docstore/mongodocstore/mongo.go

```diff
@@ -125,6 +125,11 @@ type Options struct {
 	// The name of the field holding the document revision.
 	// Defaults to docstore.DefaultRevisionField.
 	RevisionField string
+	// Whether Query.Update writes a new revision into the updated documents.
```

Comment:
We should remind the user that this option should be used on the collections where there is no revision at all. It doesn't make sense to have this option on with revision field in the data because the user will end up with staled revision after doing an update query.

Reply:
Changed the "if" to an "if and only if". Is that what you mean? Maybe suggest the doc you want to see?

Reply:
Yes, I think this is good enough.